controllers/topics-controller.js

```diff
@@ -54,4 +55,13 @@ exports.postArticleComment = (req,res,next) => {
         res.status(200).send({ comment });
     })
     .catch(next);
+};
+
+exports.deleteComment = (req,res,next) => {
+    const { comment_id } = req.params;
+    Promise.all([checkExists('comments', 'comment_id', comment_id), removeCommentByID(comment_id)])
+    .then(() => {
```

Comment:
You dont need to use a separate query to check if the comment exists here, this can all be done in the removeCommentById model.